docs/en/observability/apm/view-and-analyze/inventory.asciidoc

```diff
@@ -0,0 +1,90 @@
+[[inventory]]
+= Inventory
+
+++++
+<titleabbrev>Inventory</titleabbrev>
+++++
+
+Inventory provides a single place to observe the status of your entire ecosystem of hosts, containers, and services at a glance, even just from logs. From there, you can monitor and understand the health of your entities, check what needs attention, and start your investigations.
+
+NOTE: The new Inventory requires the Elastic Entity Model (EEM). To learn more, refer to <<elastic-entity-model>>.
+
+[role="screenshot"]
+image::./images/inventory-catalog.png[Inventory catalog]
+
+Inventory is currently available for hosts, containers, and services, but it will scale to support all of your entities.
+
+The EEM currently supports the inventory experience (as identified by host.name, service.name, and container.id) located in data identified by the following index patterns:
+
+**Hosts**
+
+Where `host.name` is set in `metrics-*`, `logs-*`, `filebeat-*`, and `metricbeat-*`
+
+**Services**
+
+Where `service.name` is set in `filebeat*`, `logs-*`, `metrics-apm.service_transaction.1m*`, and `metrics-apm.service_summary.1m*`
+
+**Containers**
+
+Where `container.id` is set in `metrics-*`, `logs-*`, `filebeat-*`, and `metricbeat-*`
+
+Inventory allows you to:
+
+- Filter for your entities to provide a high-level view of what you have leveraging your own tags and labels
+- Drill down into any host, container, or service to help you understand performance
+- Debug resource bottlenecks with your service caused by their containers and the hosts they run on.
+- Easily discover all entities related to the host, container or service you are viewing by leveraging your tags and labels
+
+[float]
+[[explore-your-entities]]
+== Explore your entities
+
+. In your Observability project, go to **Inventory** to view all of your entities.
++
+When you open the Inventory for the first time, you'll be asked to enable the EEM. Once enabled, the Inventory will be accessible to anyone with the appropriate privileges.
++
+NOTE: The Inventory feature can be completely disabled using the `observability:entityCentricExperience` flag in **Stack Management**.
+
+. In the search bar, search for your entities by name or type, for example `entity.type:service`.
+
+For each entity, you can click the entity name and get a detailed view. For example, for an entity of type service, you get the following details:
+
+- Overview
+- Transactions
+- Dependencies
+- Errors
+- Metrics
+- Infrastructure
+- Service Map
+- Logs
+- Alerts
+- Dashboards
+
+[role="screenshot"]
+image::./images/inventory-entity-detailed-view.png[Inventory detailed view]
```

Comment:
Same comment about this screenshot. It needs to be cropped to remove the nav pane.